wandb/integration/keras/callbacks/gradient_logger.py

```diff
@@ -0,0 +1,84 @@
+import wandb
+from typing import Optional, Sequence
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import models, callbacks
+
+from ..keras import patch_tf_keras, _GradAccumulatorCallback, _CustomOptimizer
+
+
+tf_logger = tf.get_logger()
+
+patch_tf_keras()
+
+
+class WandbGradientLogger(tf.keras.callbacks.Callback):
```

Comment:
I was not able to use it, 

trying to do this: `WandbGradientLogger(training_data=x_train[:1000], input_shape=(None, 784))`

Reply:
Also not sure if Keras users will use it separately or should it be part of the metric logger class. Maybe we have have this class but will be invoked from the `WandbMetricsLogger`. But for now we can keep it as it is.